### DISCUSSION TITLE
Need help with ordering menu item

### DISCUSSION THREAD

**Discussion by nyue** [thegeeklab/hugo-geekdoc#117] [Q&A] [answered]
I am having difficulty putting together the information regarding Taxonomy and Front Matter in ordering my menu item by e.g. date or weight

I have the following layout. Where do I place the weigh/date information so that I can sort my release notes in reverse chronological order i.e. 1.5.2 is the first item

![MenuItemOrderingQuestion](https://user-images.githubusercontent.com/486172/117872058-0c1a1e00-b253-11eb-84d6-bdd9d3604d73.jpg)

**Comment by xoxys** [comment]
Please use GitHub Discussions for such topics. I'll transfer this issue.

**Comment by xoxys** [comment, marked as answer]
@nyue have you already read the [documentation](https://geekdocs.de/usage/menus/#file-tree-menu)? The file-tree menu will order menu items alphabetically or by weight. Ordering by date is not possible. The `weight` need to be set in the front matter of every file that should be moved to another position, see https://github.com/thegeeklab/hugo-geekdoc/blob/main/exampleSite/content/features/_index.html

**Reply by nyue** [reply]
Thanks. Got it working.
